backend/types.ts

```diff
@@ -116,8 +116,15 @@ export type ShiftBulkRequestDTO = {
   recurrenceInterval: RecurrenceInterval;
 };
 
+export type ShiftDataWithoutPostingId = Omit<ShiftBulkRequestDTO, "postingId">;
```

Comment:
Feel free to reword but I suggest something along the lines of this so that there no confusion on whether the type refers to a single shift or not :)

```suggestion
export type ShiftBulkRequestWithoutPostingId = Omit<ShiftBulkRequestDTO, "postingId">;
```